modules/rational-modeline.el

```diff
@@ -0,0 +1,158 @@
+;;;; rational-modeline.el --- rational module for setting up the modeline  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2022
+;; SPDX-License-Identifier: MIT
+
+;; Authors: System Crafters Community, David "System Crafters" Wilson, Erik Lundstedt
+
+;; Commentary
+
+;;
+
+;;; Code:
+
+(defcustom test 'nil
+  "something here"
+  :group 'rational
+  :set (lambda (sym val)
+         (set-default sym val)
+         (when (equal test t)
+           (counsel-find-file)
```

Comment:
This shouldn't be in a productive pull request. Please remove the test code.

Reply:
Agree this should also be removed.